templates/openapi-generator/api/api.mustache

```diff
@@ -39,7 +39,7 @@ export class {{classname}}RequestFactory extends BaseAPIRequestFactory {
      * {{&summary}}
      {{/summary}}
      {{#allParams}}
-     * @param {{paramName}} {{description}}
+     * @param {{paramName}} {{&description}}
```

Comment:
To prevent unnecessary special characters escaping after running `yarn build:openapi` with latest OpenAPI 3 spec.  (Generated code changes for latest OpenAPI 3 spec will be in new PRs)